### ISSUE TITLE
how can i visual a model which not is a Classifier model?

### ISSUE THREAD

**Issue by lvguangtao** [jacobgil/pytorch-grad-cam#154]

**Comment by jacobgil** [comment]
This will require modifyin the loss function in base_cam.py.

I'm planning to allow customizing this by passing your own loss function, but in the meanwhile you will have to modify the `get_loss` function in base_cam.py
